presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1950,6 +1950,43 @@ private void testBucketedExecution(Session session)
                 "SELECT custkey, COUNT(*) FROM orders GROUP BY custkey");
     }
 
+    @Test
+    public void testBucketedFilterExecution()
+    {
+        testBucketedFilterExecution(getSession());
+    }
+
+    private void testBucketedFilterExecution(Session session)
+    {
+        try {
```

Comment:
- let's make this test run the same queries as the other test

```
        getQueryRunner().execute("CREATE TABLE orders_bucketed WITH (bucket_count = 11, bucketed_by = ARRAY['orderkey']) AS " +
                "SELECT * FROM orders");

        try {
            assertQuery("SELECT * FROM orders_bucketed WHERE orderkey = 100", "SELECT * FROM orders WHER orderkey = 100");
            assertQuery("SELECT * FROM orders_bucketed WHERE orderkey = 100 OR orderkey = 101", "SELECT * FROM orders WHERE orderkey = 100 OR orderkey = 101");
            // TODO add more tests
        }
        finally {
            assertUpdate("DROP TABLE orders_bucketed");
        }
```

Reply:
Done